app/Models/Post.php

```diff
@@ -195,7 +195,6 @@ public function toBlinkPayload()
             'caption' => $this->text,
             'text' => $this->text,
             'status' => $this->status,
-            'remote_addr' => $this->remote_addr,
```

Comment:
I think this spot and the Quasar one below are where we might want to return `000.000.00.0` too

Reply:
ah you're totally right @katiecrane thank you!